src/hip/hip_build_utils.cpp

```diff
@@ -174,20 +171,20 @@ static fs::path HipBuildImpl(boost::optional<TmpDir>& tmp_dir,
 #endif
 
     // hip version
-    params +=
-        std::string(" -DHIP_PACKAGE_VERSION_FLAT=") + std::to_string(HIP_PACKAGE_VERSION_FLAT);
-
-    params += " ";
+    params += " -DHIP_PACKAGE_VERSION_FLAT=" MIOPEN_STRINGIZE(HIP_PACKAGE_VERSION_FLAT) " ";
```

Comment:
`HIP_PACKAGE_VERSION_FLAT` expands to a parenthesized expression. Please use std::to_string. Thanks.